video_xblock/video_xblock.py

```diff
@@ -300,18 +299,17 @@ def player_state(self):
         transcripts_object = {
             trans['lang']: {'url': trans['url'], 'label': trans['label']}
             for trans in transcripts
-        }
-        return {
-            'current_time': self.current_time,
-            'muted': self.muted,
-            'playback_rate': self.playback_rate,
-            'volume': self.volume,
-            'transcripts': transcripts,
-            'transcripts_enabled': self.transcripts_enabled,
-            'captions_enabled': self.captions_enabled,
-            'captions_language': self.captions_language or course.language,
-            'transcripts_object': transcripts_object
-        }
+            }
+        result = dict()
+        result['captionsLanguage'] = self.captions_language or course.language
+        result['transcripts'] = transcripts
```

Comment:
As far as I can see following for loop will overwrite `result['captionsLanguage']` and `result['transcripts']`.